conduit/plugins/processors/filterprocessor/filter_processor_bench_test.go

```diff
@@ -24,6 +24,23 @@ func blockData(addr sdk.Address, numInner int) (block data.BlockData, searchTag
 				SignedTxnWithAD: sdk.SignedTxnWithAD{
 					SignedTxn: sdk.SignedTxn{
 						AuthAddr: addr,
+						Txn: sdk.Transaction{
+							Header: sdk.Header{
+								Group: sdk.Digest{1},
+							},
+						},
+					},
+				},
+			},
+			{
+				SignedTxnWithAD: sdk.SignedTxnWithAD{
+					SignedTxn: sdk.SignedTxn{
+						AuthAddr: addr,
```

Comment:
This is set in both transactions, are you trying to show that including the group is faster?

Reply:
oh. I should remove this in this txn. I'm adding a 2nd txn for benchmarking txn filtering that returns the txn group.